### PR TITLE
re-adds some lost accesses (QM, Atmos Tech)

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

got lost in https://github.com/Skyrat-SS13/Skyrat-tg/pull/13751

updates some instances of `access = ` to `minimal_access = `

## How This Contributes To The Skyrat Roleplay Experience

fix

## Changelog
:cl:
fix: QMs and atmos techs once more have the access they temporarily lost
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/datums/id_trim/jobs.dm [hunk 1]
```diff
@@ -29,6 +29,11 @@
 
 	access |= ACCESS_WEAPONS
 
+/datum/id_trim/job/atmospheric_technician/New()
+	. = ..()
+
+	minimal_access |= ACCESS_ENGINE_EQUIP
```
Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/datums/id_trim/jobs.dm:35]: is minimal the correct way to do this?

Comment (Gandalf2k15) [inline modular_skyrat/master_files/code/datums/id_trim/jobs.dm:35]: is minimal the correct way to do this?

Comment (tf-4) [reply modular_skyrat/master_files/code/datums/id_trim/jobs.dm:35]: yes

Comment (Gandalf2k15) [reply modular_skyrat/master_files/code/datums/id_trim/jobs.dm:35]: Update them all to this then?

**Pull request merged**